packages/web-config-server/src/apiV1/organisationUnit.js

```diff
@@ -18,6 +18,21 @@ export default class extends RouteHandler {
   async buildResponse() {
     const { includeCountryData } = this.query;
     const project = await this.fetchProject();
+
+    // If this is a project entity, get entitiesWithAccess and set them on the entity
+    if (this.entity.isProject()) {
+      const orgUnits = await this.entity.getNearestOrgUnitDescendants();
+
+      const entitiesWithAccessInfo = await Promise.all(
+        orgUnits.map(async orgUnit => ({
+          ...orgUnit,
```

Comment:
Did you consider that option of just storing the bounds, rather than full entity details? All good if you did and prefer the whole entity

Reply:
Yeah I did look at it but I decided I like this better as it made sense to me that way

Reply:
Sweet - just that typo to go!